demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -236,9 +233,12 @@ const productsFragment = gql`
             id
             title
         }
-        variants {
-            image
+        colors {
             name
+            colorHexCode
```

Comment:
This changed in the base branch, you probably need a rebase

Reply:
hm, this was added in this PR, and I remember changing that. But somehow it got lost....